scripts/us_census/acs5yr/subject_tables/jsonSpec.md

```diff
@@ -0,0 +1,203 @@
+The `_DEFAULT` key is **mandatory**.
+
+```json
+ "populationType":{
+		"_DEFAULT": "Person",
+		"Total household population": "Household",
+		"Median household income of householders": "Household"
+   }
+
+```
+
+In this example, which is based on the subject table S2702, the `_DEFAULT` value for the `populationType` property is set as `Person`. However, to specify other population types in the dataet, we define the value for the `populationType` property, based on the column tokens. For example, if the column name contains the token "Total household population", the `populationType` property is assigned the value `Household`.
+
+> **#TODO: Describe the notion of column tokens**
+
+
+#### `measurement`
+The `measurement` section of the spec defines the stat-var properties that are related to a measurement, specified through column tokens.
+The `_DEFAULT` key is **mandatory**.
+
+```json
+   "measurement":{
+		"Margin of Error": {
+			"measuredProperty": "count",
+			"statType": "marginOfError"
+		},
+		"Median age (years)": {
+			"measuredProperty": "age",
+			"statType": "medianValue",
+			"unit": "Years"
+		},
+		
+		"Median household income of householders": {
+			"measuredProperty": "income",
+			"statType": "medianValue",
+			"unit": "USDollar"
+		},
+
+		"_DEFAULT": {
+			"measuredProperty": "count",
+			"statType": "measuredValue"
+		}
+   }
+
+```
+
+#### `pvs`
+The `pvs` section describes the additional constraint properties that are added to define a stat-var based on column tokens.
+
+```json
+  "pvs":{
+		"healthInsurance": {
+			"Total Uninsured": "NoHealthInsurance",
+			"Uninsured Population": "NoHealthInsurance",
+			"Total Uninsured MOE": "NoHealthInsurance"
+		}
+}
+
+```
+
+#### `universePVs` [optional]
+Defines a set of additional properties and values to the stat-var based on a specific `populationType` and list of constraint properties.
+
+In the example, if the `populationType` is a `Person` and the constraint property is `healthInsurance`, the additional property-values for `armedForcesStatus` and `institutionalization` are added. If there are multiple constrainProperties, the additionals pvs (dependent PVs) are added with a logical `AND` condition.
+
+```json
+  "universePvs":[
+		{
+			"populationType": "Person",
+			"constraintProperties": ["healthInsurance"],
+			"dependentPVs": {
+				"armedForcesStatus": "Civilian",
+				"institutionalization": "USC_NonInstitutionalized"
+			}
+		}
+   ]
+```
+
+
+#### `inferedSpec` [optional]
+Adds additional property-value pairs to a stat-var node based on the inference of an existing property in that stat-var node.
```

Comment:
the following may be easier to understand:
"based on the inference of an existing property" -> "based on the presence of another property"